src/admin/components/OnlineDriver/OnlineDriverTable.tsx

```diff
@@ -29,39 +21,33 @@ function OnlineDriverTable() {
     return () => clearInterval(intervalId);
   }, []);
```

Comment:
Use ESLint extension to identify warnings like adding "repo" to the dependency array "[]" here and in all other similar cases.